physics/module_mp_thompson.F90

```diff
@@ -2006,6 +2016,8 @@ subroutine mp_thompson (qv1d, qc1d, qi1d, qr1d, qs1d, qg1d, ni1d,    &
       odt = 1./dt
       odts = 1./dtsave
       iexfrq = 1
+! transition of terminal velocity from cloud ice to snow
```

Comment:
My suggestion for comment is:  Transition value of coefficient matching at crossover from cloud ice to snow.

Reply:
FIne with me.

Reply:
Done as suggested.